client/src/featureform/enums.py

```diff
@@ -51,6 +51,7 @@ class OperationType(Enum):
 @dataclass
 class SourceType(str, Enum):
     PRIMARY_SOURCE = "PRIMARY"
+    DIRECTORY = "DIRECTORY"
```

Comment:
what's the diff between primary and directory? is primary just a regular file?

Reply:
coming back to this, I feel like directory would be a type of primary source

Reply:
yeah same, not a good format for this atm though